src/binder_cell_info.c

```diff
@@ -102,6 +102,24 @@ binder_cell_info_int_format(
     }
 }
 
+static
+const char*
+binder_cell_info_int64_format(
+    guint64 value,
+    const char* format)
+{
+    if (value == OFONO_CELL_INVALID_VALUE_INT64) {
+        return "";
+    } else {
+        static GUtilIdlePool* binder_cell_info_pool = NULL;
```

Comment:
static `binder_cell_info_pool` was meant to be moved to the top (file) level when it's needed from more than one place. That would be slightly more efficient (veeery slightly, I admit)

Reply:
Moved the static variable to top of the file next to another static variable.